lib/apiservers/portlayer/restapi/handlers/containers_handlers.go

```diff
@@ -118,6 +118,10 @@ func (handler *ContainersHandlersImpl) CreateHandler(params containers.CreatePar
 		Key:      pem.EncodeToMemory(&privateKeyBlock),
 		LayerID:  *params.CreateConfig.Image,
 		RepoName: *params.CreateConfig.RepoName,
+		Resources: executor.Resources{
+			NumCPUs: *params.CreateConfig.NumCpus,
+			Memory:  *params.CreateConfig.Memory,
```

Comment:
@jzt I think @caglar10ur is requesting units be part of the field name rather than we use MB vs MiB.